hack/run-test.sh

```diff
@@ -92,6 +92,7 @@ main() {
     if [ "$ONLY_CLI_PRINT" = true ]; then
         print "Skipping execution as requested by user ... "
     else
+        trap '{ echo "Running cleanup test suite"; ginkgo test/e2e/performanceprofile/functests/Z_deconfig --flake-attempts=2 --junit-report=report.xml; }' EXIT SIGINT SIGTERM SIGSTOP
```

Comment:
Please be more specific:
`Deleting node inspector pod`

Reply:
@Tal-or I chose to write this in a general way, as in the future Z_deconfig might be used to do more than cleaning node inspector.

Reply:
When it does, we can always rephrase